composer.json

```diff
@@ -20,7 +20,7 @@
     ],
     "require": {
         "php":                       "7.4.*",
-        "zendframework/zend-code":   "^3.4.0",
+        "zendframework/zend-code":   "^3.4.1",
```

Comment:
zendframework/zend-code started outputting "Package zendframework/zend-code is abandoned, you should avoid using it. Use laminas/laminas-code instead". It may be wise to change the package name at the same time.

Reply:
Laminas was migrated tonight (happy birthday, @laminas!): if you want to help, please send a patch 👍

Reply:
I think their migration is still ongoing. The packages are not yet listed on packagist https://packagist.org/packages/laminas/ and composer won't find them. I'll create the PR once it's done.